### DISCUSSION TITLE
how to configure a link to a page on the website that is not in the TOC

### DISCUSSION THREAD

**Discussion by chris-habasinski** [GitbookIO/community#627] [3. Git Sync]
Hi everyone

I have the github sync integration enabled for my space.
I am trying to create a hyperlink on a page which points to a generic page that is used when no entry yet exists for that link.
Let me explain with an example.
For an existing documented feature I have - this works:
```
[link to documented feaure](../path/to/exists.md)
```
For an existing undocumented feature I want to have:
```
[link to undocumented feature](../extras/tbd.md)
```
`extras/tbd.md` exists BUT I do not want it to show up in my TOC.

If I create a custom SUMMARY.md file that excludes `extras/tbd.md`, then the file does not get copied to gitbook and the link is broken.

In general, I have been unable to create a hyperlink to any resource other than a markdown page without the link pointing to my github repository.

I could use the `{% file xxxxx %}` construct  and make the destination an image, but the link is not presented inline.

I have tried various other methods, like linking ot the `.gitbook/assets` folder, trying to use `content-ref` but nothing allows me to hyperlink to a webpage that is not in the TOC.

Any ideas? I hope I am missing something simple.

Thanks

Chris

**Comment by aikoii** [comment]
Hey @chris-habasinski 👋! Currently, in GitBook, any page you want to link to must be listed in the `SUMMARY.md` file to be accessible. There isn't a way to create a hyperlink to a page without it appearing in the TOC or `SUMMARY.md.` Sorry this isn't ideal 🌸

**Comment by siscia** [comment]
Is this still the case?

**Comment by chris-habasinski** [comment]
Hi @siscia 
This has been addressed, although I have not tested it yet.
See: https://docs.gitbook.com/content-editor/editor/content-structure/content-in-a-space
Chris
